workspace_definitions.bzl

```diff
@@ -102,3 +103,8 @@ def rules_foreign_cc_dependencies(
             "@rules_foreign_cc//tools/build_defs:preinstalled_cmake_toolchain",
             "@rules_foreign_cc//tools/build_defs:preinstalled_ninja_toolchain",
         )
+
+    native.register_toolchains(
```

Comment:
Please move it into /examples/WORKSPACE, then it will work fine with tests.
We should not have test toolchains in rule initialization code.

Reply:
Done.